codes/classical/bits/raptor.yml

```diff
@@ -0,0 +1,50 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: raptor
+physical: bits
+logical: bits
+
+name: 'Raptor code'
+introduced: '\cite{manual:{A. Shokrohalli. \emph{Raptor Codes}. IEEE Transactions on Information Theory, 2006}}'
```

Comment:
\cite{doi:10.1109/TIT.2006.874390}
CHange other link to preferably use DOI, then arXiv ref, and finally manual. See template for details.